src/dune_engine/action_exec.ml

```diff
@@ -542,6 +542,36 @@ type input =
   ; action : Action.t
   }
 
+module Install_root = struct
+  let path_sep = if Sys.win32 then ';' else ':'
+
+  let ( ^/ ) = Filename.concat
```

Comment:
just `/` should be fine for this.

Reply:
Sorry I should have been clearer. I meant / as an operator rather than */ so that it is more uniform with how we do it elsewhere.

Reply:
I understand now. You meant use `(/)` instead of `(^/)`. I made that change in the latest push.